README.rst

```diff
@@ -151,13 +151,13 @@ Past and current funding and support for HPX is listed `here
      :target: https://doi.org/10.5281/zenodo.598202
      :alt: Latest software release of HPX
 
-.. |codacy| image:: https://app.codacy.com/project/badge/Grade/0b8cd5a874914edaba67ce3bb711e688
+.. |codacy_quality| image:: https://app.codacy.com/project/badge/Grade/0b8cd5a874914edaba67ce3bb711e688
      :target: https://app.codacy.com/gh/STEllAR-GROUP/hpx/dashboard
      :alt: HPX Code Quality Assessment
 
-.. |coveralls| image:: https://coveralls.io/repos/github/STEllAR-GROUP/hpx/badge.svg
-     :target: https://coveralls.io/github/STEllAR-GROUP/hpx
-     :alt: HPX coverage report
+.. |codacy_coverage| image:: https://app.codacy.com/project/badge/Coverage/0b8cd5a874914edaba67ce3bb711e688    
+  :target: https://app.codacy.com/gh/STEllAR-GROUP/hpx/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_coverage
```

Comment:
I think you can remove all the `?key=value&...` parameters on this url.

Reply:
Codacy gave this to me as-is, but you are completely right in that it doesn't make a single difference.